src/Runner.Listener/BrokerMessageListener.cs

```diff
@@ -42,7 +42,7 @@ public override void Initialize(IHostContext hostContext)
             _brokerServer = HostContext.GetService<IBrokerServer>();
         }
 
-        public async Task<Boolean> CreateSessionAsync(CancellationToken token)
+        public async Task<Constants.Runner.CreateSessionResult> CreateSessionAsync(CancellationToken token)
```

Comment:
Do you mind moving the enum to a different place? This method return signature looks funny because the result value by definition is not a constant (i.e. it can be one of multiple different values).

Maybe the Runner.Listener namespace would be a good place to define the enum since usage is localized to that namespace.